extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -929,6 +932,17 @@ describes.repeated(
               expect(list.layoutCallback()).eventually.rejectedWith(errorMsg);
             });
 
+            it('"amp-script:" uri should skip rendering and emit an error', () => {
+              toggleExperiment(win, 'protocol-adapters', true);
```

Comment:
Should this be reset at the end of this block?

Reply:
Yes! It only happens to work right now since it is the only amp-script test in its section.

Reply:
fixed!